### PR TITLE
LPS-139253 Test Autom. Content performance side panel

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**Ticket**:
[LPS-139253](https://issues.liferay.com/browse/LPS-139253)
[LPS-113910](https://issues.liferay.com/browse/LPS-113910)

**Motivation**:
Test automation: "Content performance metrics are shown in the side panel"

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
Hey @Tim-Cao ! Could you please review this PR ?

**Comment by Luizcarlosqueiroz** [comment]
Hey @CarlosBrichete! Now that you're back, could you review this PR?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 1]
```diff
@@ -400,30 +400,177 @@ definition {
...
+				contentType = "Web Content Article",
+				displayPageTemplateEntryName = "Display Page Name",
+				groupName = "Test Site Name",
+				subType = "Basic Web Content");
+
+			DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
+
+			DisplayPageTemplatesAdmin.gotoDisplayPage(displayPageName = "Display Page Name");
+
+			Button.clickPublish();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:423]: PageEditor.clickPublish();

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 2]
```diff
@@ -400,30 +400,177 @@ definition {
...
+
+			DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:425]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 3]
```diff
@@ -400,30 +400,177 @@ definition {
...
+				item = "Info",
+				tableEntry = "WC Title");
+		}
 
+		task ("Check if Metric panel is closed and Info panel is openned") {
+			AssertTextNotPresent(
+				locator1 = "ContentDashboard#INFO_PANEL_TITLE",
+				value1 = "Content Performance");
+
+			AssertTextPresent(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:451]: AssertTextEquals

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 4]
```diff
@@ -400,30 +400,177 @@ definition {
...
+		}
+
+		task ("Open Info panel") {
+			LexiconTable.clickEllipsisItem(
+				item = "Info",
+				tableEntry = "WC Title");
+		}
 
+		task ("Check if Metric panel is closed and Info panel is openned") {
+			AssertTextNotPresent(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:447]: AssertTextNotEquals

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 5]
```diff
@@ -400,30 +400,177 @@ definition {
...
+			AssertTextPresent(
+				locator1 = "ContentDashboard#INFO_PANEL_TITLE",
+				value1 = "WC Title");
+		}
 	}
 
 	@description = "LPS-113910: When the user has NOT permissions for editing and viewing the web content and the web content has a display page template associated, the View metrics option is not displayed"
-	@ignore = "true"
 	@priority = "4"
 	test MetricsUserWithoutPermissionsDPT {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:459]: Add property custom.properties = "jsonws.web.service.paths.excludes=";

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 6]
```diff
@@ -400,30 +400,177 @@ definition {
...
+				contentType = "Web Content Article",
+				displayPageTemplateEntryName = "Display Page Name",
+				groupName = "Test Site Name",
+				subType = "Basic Web Content");
+
+			DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
+
+			DisplayPageTemplatesAdmin.gotoDisplayPage(displayPageName = "Display Page Name");
+
+			Button.clickPublish();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:517]: PageEditor.clickPublish();

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 7]
```diff
@@ -400,30 +400,177 @@ definition {
...
+
+			DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:519]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 8]
```diff
@@ -400,30 +400,177 @@ definition {
...
+				userLoginFullName = "Test Test");
+		}
+
+		task ("Delete role crated via JSONWS") {
+			JSONRole.deleteRole(roleTitle = "Content Dashboard Viewer");
+		}
+
+		task ("Delete user created") {
+			JSONUser.tearDownNonAdminUsers();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:552]: Move to teardown block

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 9]
```diff
@@ -400,30 +400,177 @@ definition {
...
+			AssertNotVisible(locator1 = "ContentPerformance#CONTENT_PERFORMANCE_BUTTON");
+		}
+
+		task ("Longin with Test Tes user to allow teardown") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+		}
+
+		task ("Delete role crated via JSONWS") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:546]: Delete role via JSONWS

**Review by Tim-Cao — COMMENTED** [review]
Hi @Luizcarlosqueiroz, Could you update according to comments? Thanks

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**